b2sdk/raw_api.py

```diff
@@ -143,7 +147,14 @@ def get_download_authorization(
         pass
 
     @abstractmethod
-    def get_file_info(self, api_url, account_auth_token, file_id):
+    def get_file_info_by_id(self, api_url: str, account_auth_token: str,
+                            file_id: str) -> Dict[str, Any]:
+        pass
+
+    @abstractmethod
+    def get_file_info_by_name(
+        self, download_url: str, account_auth_token: str, bucket_name: str, file_name: str
+    ) -> CaseInsensitiveDict:
```

Comment:
this ties raw_api interface with `requests`, but soon we will move to `curl` and then having requests as a dependency in the interface will be awkward. Can we please use `Dict[str, str]` here to avoid the dependency in the interface?